xla/service/gpu/fusions/mlir/BUILD

```diff
@@ -107,6 +108,7 @@ cc_library(
 xla_cc_test(
     name = "elemental_hlo_to_mlir_test",
     srcs = ["elemental_hlo_to_mlir_test.cc"],
+    tags = tf_gpu_tests_tags(),
```

Comment:
Curious if this change is necessary, it's not totally obvious to me that this test needs a GPU to run

Reply:
This was added so that CI can pick up this test to run. IMO it does not require a gpu to run. Adding a "cpu" tag should be sufficient.